lorem.html

```diff
@@ -19,35 +19,13 @@
     <link href="styles/Main.css" rel="stylesheet">
     <link rel="manifest" href="site.webmanifest" />
     <meta name="msapplication-TileColor" content="#da532c" />
-    <meta name="theme-color" content="#ffffff" />\
+    <meta name="theme-color" content="#ffffff" />
     <style>
-      h1{ 
-        background: rgb(2,0,36);
-        background: linear-gradient(180deg, rgba(2,0,36,1) 0%, rgba(9,121,110,0.36458333333333337) 24%, rgba(23,111,163,1) 100%);
-      }
-      h1,h2,h3,h4,h5,h6 {
-        font-family: 'Space Mono', monospace;
-      }
-      p {
-        font-family: 'Asap', sans-serif;
-      }
-      body a{
-        text-decoration: none;
-      }
-      body a:hover{
-        text-decoration: underline;
-      }
       img{
         margin: 1em;
       }
       footer{
-        font-size: 75%;
-        text-align: center;
-        float: none;
```

Comment:
This just makes the footer non-floating.  To make it clear *other* floated elements, use `clear: both`